netneurotools/plotting.py

```diff
@@ -416,6 +416,58 @@ def plot_fsaverage(data, *, lhannot, rhannot, order='lr', mask=None,
     -------
     brain : surfer.Brain
         Plotted PySurfer brain
+
+    Notes
+    -----
+    -  Shown here the minimal example to plot on the fsaverage surface for
+       Lausanne and Schaefer parcellation. Optionally, fetch the annotation
+       files using :py:func:`netneurotools.datasets.fetch_fsaverage`, here
+       with ``version="fsaverage"``.
+
+    -  For Lausanne, use
+       :py:func:`netneurotools.datasets.fetch_cammoun2012`
+       and ``plot_lausanne_surf()`` below.
+
+        -  Normally Lausanne connectomes have right hemisphere before left
+           hemisphere, thus ``order="RL"``
+
+    -  For Schaefer, use
+       :py:func:`netneurotools.datasets.fetch_schaefer2018`
+       and ``plot_schaefer_surf()`` below.
```

Comment:
Based on my suggestion below maybe we could clean this up / simplify this a bit. (Also, I'd prefer to not use bullet points in the Notes section if possible—just line break should be fine!).